cli/spec.py

```diff
@@ -114,6 +114,38 @@ def override_default_values(clg_args, sub_parser_options):
         _check_required_arguments(sub_parser_options, clg_args)
 
 
+def post_process_command_args(clg_args, app_settings_dir):
+    """
+    Transforms some of arguments after they have been read.
+
+    :param clg_args: Dictionary based on cmd-line args parsed by clg
+    :param app_settings_dir: path to the base directory holding the
+        application's settings. App can be provisioner\installer\tester
+        and the path would be: settings/<app_name>/
+    """
+
+    # post process topology
+    if 'topology' in clg_args and clg_args['topology'] is not None:
+        topology_dict = {}
+        for topology_item in clg_args['topology'].split(','):
+            if '_' in topology_item:
+                number, node_type = topology_item.split('_')
+            else:
+                raise exceptions.IRConfigurationException(
+                    "Topology node should be in format  <number>_<node role>. "
+                    "Current value: '{}' ".format(topology_item))
+            if int(number) > 1 and node_type[-1] == 's':
```

Comment:
This could invite bugs with english grammer :smile: 
Let's stick with matching the file name exactly.